test/functional/ex_cmds/menu_spec.lua

```diff
@@ -36,3 +36,37 @@ describe(':emenu', function()
   end)
 
 end)
+
+describe('emenu Edit.Paste while in commandline', function()
+    before_each(clear)
+
+    it('ok', function()
+        local screen = require('test.functional.ui.screen').new()
```

Comment:
Can you change the `new()` call to `new(40, 8)` and adjust the test below? Most tests (like this one) don't need such a big space, so this compacts the tests a bit for readability.